runtime/src/main/cpp/math/fmodf.cpp

```diff
@@ -1,6 +1,10 @@
 #include <math.h>
 #include <stdint.h>
 
+#ifdef KONAN_WASM
+#include "../Common.h"
```

Comment:
Please do't do includes by relative paths.
Also add RUNTIME_USED in separate ifdef, code is unreadable this way.